extension/main.js

```diff
@@ -55,12 +55,13 @@
 	function handleNotificationClicked(notificationId) {
 		const url = window.GitHubNotify.settings.get(notificationId);
 		if (url) {
-			fetch(url).then(res => res.json()).then(json => {
+			window.GitHubNotify.request(url).then(res => res.json()).then(json => {
 				openTab(json.html_url);
 			}).catch(() => {
 				openTab(window.GitHubNotify.getTabUrl());
 			});
 		}
+		chrome.notifications.clear(notificationId);
```

Comment:
Stealthy fix that closes the notification when it's clicked on by the user, which I required for my use of the extension, haha.